addons/pkg-npm/jaxrs/src/main/java/org/commonjava/indy/pkg/npm/jaxrs/NPMContentAccessResource.java

```diff
@@ -145,13 +159,27 @@ public Response doHead(
     public Response doHead(
                     final @ApiParam( allowableValues = "hosted,group,remote", required = true ) @PathParam( "type" ) String type,
                     final @ApiParam( required = true ) @PathParam( "name" ) String name,
-                    final @PathParam( "packageName" ) String packageName, final @PathParam( "version" ) String version,
+                    final @PathParam( "packageName" ) String packageName,
+                    final @ApiParam( allowableValues = CROSS ) @PathParam( "version" ) String version,
```

Comment:
@jdcasey This will be same rest path params with /{packageName}/{version} if remove CROSS, so I could only make the '-' fixed here to distinguish this api form others. I've no other better idea, do you have any commnet?